client-dotnet/src/EvoMaster.Instrumentation/Heuristic/TruthnessUtils.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using EvoMaster.Instrumentation.Heuristic;
+
+public class TruthnessUtils {
+
+    /**
+     * scales to a positive double value to the [0,1] range
+     *
+     * @param v a non-negative double value
+     * @return
+     */
+    public static double NormalizeValue(double v) {
+        if (v < 0) {
+            throw new ArgumentException("Negative value: " + v);
+        }
+
+        if(!Double.IsFinite(v) || v == Double.MaxValue){
+            return 1d;
+        }
+
+        //normalization function from old ICST/STVR paper
+        double normalized = v / (v + 1d);
+
+        //assert normalized >= 0 && normalized <= 1; TODO
```

Comment:
in C#, can't this be handled with `Debug.Assert` ?

Reply:
I handled this with Trace.Assert which works in release mode unlike Debug.Assert which works in debug mode. Is it ok?